_delphi_utils_python/delphi_utils/archive.py

```diff
@@ -89,6 +94,42 @@ def diff_export_csv(
         after_df_cmn.loc[~(same_mask.all(axis=1)), :],
         after_df.loc[added_idx, :])
 
+
+def run_module(archive_type: str,
```

Comment:
Highly suspicious. This change is already in main as [530aba32](https://github.com/cmu-delphi/covidcast-indicators/commit/530aba322a9294d3cec559121afb23a9535c415e); what is it doing in this PR?

Reply:
Mmm, yeah from merging in main. What is your preference for fixing?